flink-clients/src/main/java/org/apache/flink/client/program/ClusterClient.java

```diff
@@ -65,9 +65,7 @@ default void close() throws Exception {
 	/**
 	 * Shut down the cluster that this client communicate with.
 	 */
-	default void shutDownCluster() {
-		throw new UnsupportedOperationException();
-	}
+	void shutDownCluster();
```

Comment:
What about classes that extend ClusterClient ?

Reply:
Within Flink scope we implement this method for all cluster client and I think the consensus is that `ClusterClient` is an internal concept at least for now. Besides, every implementation of `ClusterClient` should think of whether or not and how to shut down its associated cluster.

However, although it makes no sense add a default implementation just `throw new UnsupportedOperationException()`, if you insist I can add back such default implementation just for outside implementation pass compiler checking.

Reply:
Oh, my misunderstanding. I took it as new method, and thought other classes has no implementation for this method.